aws-amplify/amplify-android
# feat(apollo): Use HTTP headers instead of query parameters for authorization

*Issue #, if available:*

*Description of changes:*
AppSync recently added the ability to use HTTP headers instead of query parameters to authorize the websocket connection request. This is now the recommended way to connect (and new AppSync instances will eventually stop accepting query parameters).

This change switches the `apollo-appsync` library to use this new method of authorization.

*How did you test these changes?*
- Added unit tests
- Manual test passes
- Integration tests still pass

*Documentation update required?*

*General Checklist*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
